src/OpenAi.php

```diff
@@ -93,25 +125,52 @@ public function engine($engine)
      */
     private function sendRequest(string $url, string $method, array $opts = null)
     {
-        $curl_info = [
-            CURLOPT_URL => $url,
-            CURLOPT_RETURNTRANSFER => true,
-            CURLOPT_ENCODING => '',
-            CURLOPT_MAXREDIRS => 10,
-            CURLOPT_TIMEOUT => 0,
-            CURLOPT_FOLLOWLOCATION => true,
-            CURLOPT_HTTP_VERSION => CURL_HTTP_VERSION_1_1,
-            CURLOPT_CUSTOMREQUEST => $method,
-            CURLOPT_POSTFIELDS => json_encode($opts),
-            CURLOPT_HTTPHEADER => $this->headers,
-        ];
+        $curl_info = [];
+        if($method == 'POST' || $method == 'GET') //custom post or get
+        {
+            $curl_info = [
+                CURLOPT_URL => $url,
+                CURLOPT_RETURNTRANSFER => true,
+                CURLOPT_ENCODING => '',
+                CURLOPT_MAXREDIRS => 10,
+                CURLOPT_TIMEOUT => 0,
+                CURLOPT_FOLLOWLOCATION => true,
+                CURLOPT_HTTP_VERSION => CURL_HTTP_VERSION_1_1,
+                CURLOPT_CUSTOMREQUEST => $method,
+                CURLOPT_POSTFIELDS => json_encode($opts),
+                CURLOPT_HTTPHEADER => $this->headers,
+            ];
+        }
+        elseif($method == 'FILE_POST') //case for the file[default post command]
+        {
+            $curl_info = [
+                CURLOPT_URL => $url,
+                CURLOPT_RETURNTRANSFER => true,
+                CURLOPT_HTTP_VERSION => CURL_HTTP_VERSION_1_1,
+                CURLOPT_POST => true,
+                CURLOPT_POSTFIELDS => $opts,
+                CURLOPT_HTTPHEADER => $this->headers,
+            ];
+        }
+        elseif($method == 'FILE_GET') //case for the getting files[default post command]
+        {
+            $curl_info = [
+                CURLOPT_URL => $url,
+                CURLOPT_RETURNTRANSFER => true,
+                CURLOPT_HTTP_VERSION => CURL_HTTP_VERSION_1_1,
+                CURLOPT_CUSTOMREQUEST => 'GET',
+                CURLOPT_HTTPHEADER => $this->headers,
+            ];
+        }
+
         if ($opts == null) {
-            unset($curl_info[CURLOPT_POSTFIELDS]);
+            unset($curl_info["CURLOPT_POSTFIELDS"]);
```

Comment:
`CURLOPT_POSTFIELDS` is a Curl's constant. Please remove `"`'s
```suggestion
            unset($curl_info[CURLOPT_POSTFIELDS]);
```